### DISCUSSION TITLE
Not getting success message after publishing changes [Bug]

### DISCUSSION THREAD

**Discussion by iroshandezilva** [webstudio-is/webstudio-community#44] [Q&A]
Currently, once the user publishes the changes, and when it's done, there aren't any success messages "Toast/Notification." The user can only see the status by clicking the first level publish button again and seeing the "Publish" button's loading animation. I suggest having a success message after publishing changes (maybe it can be an error message) and showing the loading animation in the first-level publish button .

**Comment by kof** [comment]
So the reason is because it takes a minute, you are closing the popover and then you don't know when its done, right?

The problem is that it takes longer than you would wait, is this correct? If yes maybe the solution is to speed it up and not toast? We know we can optimize things there.

**Reply by iroshandezilva** [reply]
If we check what Webflow is doing, it provides the good experience:

1. Showing a loading button in the first level to indicate publishing happening.
2. Once publishing is finished, they show a cool animation with a rocket and change the button text to "Published" for 2 seconds.
3. Once you click publish again, you can see when the last publish happened.

![image](https://github.com/webstudio-is/webstudio/assets/55875936/0aa12d93-388b-4903-b1bc-218aab3ae7fb)

**Reply by kof** [reply]
So what is the problem you are looking to solve? Is that that we don't show that the site was published after its done in a way that lets you see that after you get there?

**Reply by kof** [reply]
<img width="291" alt="Screenshot 2023-07-11 at 12 17 45" src="https://github.com/webstudio-is/webstudio/assets/52824/a6f1f923-caf4-4663-a780-462bbd136a7b">

Is the tooltip about published info not helping?
